generators/client/templates/react/package.json.ejs

```diff
@@ -212,6 +212,7 @@
     "test": "<%= clientPackageManager %> run jest --",
     "test-ci": "<%= clientPackageManager %> run lint && <%= clientPackageManager %> run jest:update --",
     "test:watch": "<%= clientPackageManager %> run jest -- --watch",
+    "watch": "concurrently 'npm run start' npm:backend:start",
```

Comment:
Is this available for all microfrontend frameworks? If so, is it documented?

Reply:
Should available at generated project README
https://github.com/jhipster/generator-jhipster/blob/39a026ad4a5e1c27559e2b42f894a3f0935b9f09/generators/client/templates/common/README.md.jhi.client.ejs#L181